### ISSUE TITLE
Can't upgrade v2 from alpha.55 -> alpha.61 -Can't resolve 'docsearch.js'

### ISSUE THREAD

**Issue by davidkartuzinski** [facebook/docusaurus#3279]
## 🐛 Bug Report

Thank you for docusaurus and Version 2 is already incredible.

I am trying to upgrade from alpha.55 to version alpha.61. I can't tell what I am doing wrong but I am getting an error
 
```
Failed to compile
./src/theme/SearchBar/index.js
Module not found: Can't resolve 'docsearch.js' in '/Users/david/repos/erxes/docs/src/theme/SearchBar'
```

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes, I have read this.

## To Reproduce

1. In my package.json, under the dependences, change from version alpha.55, to version alpha.61
```
   "@docusaurus/core": "^2.0.0-alpha.61",
    "@docusaurus/preset-classic": "^2.0.0-alpha.61",
```
2. In the terminal, in the directory for "/docs", run `yarn install` to install upgraded version
3. Then run `yarn start`

## Expected behavior

I expected to have my installation work in the browser, and to be able to continue developing my app.

## Actual Behavior

Something went wrong:

1. I got a bunch of warnings- but I don't think this is the cause.
![Screen Shot 2020-08-13 at 18 55 40](https://user-images.githubusercontent.com/29503118/90163611-9d2c9900-dd96-11ea-95f7-829cda2b036b.png)

2. I ran `yarn start`. I got an error `ValidationError: "homePageId" is not allowed to be empty`

![Screen Shot 2020-08-13 at 18 57 55](https://user-images.githubusercontent.com/29503118/90163838-ed0b6000-dd96-11ea-974a-007e60498bd6.png)

So,  in the docusaurus.config.js, I commented out `homepageId` <-- which was in empty quotes. (I have no idea what to put in there,  I tried 'docs/'.  Either way I am able to continue with the installation. I tried with and without `docs/` (See below)

3. I ran `yarn start` again. I got a reasonable error message, `ValidationError: themeConfig.navbar.links has been renamed as themeConfig.navbar.items`  

![Screen Shot 2020-08-13 at 19 02 29](https://user-images.githubusercontent.com/29503118/90164335-8fc3de80-dd97-11ea-90aa-532d2f4ecdd4.png)

So, in the docusaurus.config.js, I changed `themeConfig.navbar.links` to `themeConfig.navbar.items`.

4. I ran `yarn start` again. I got an error `ValidationError: "navbar.items[0].activeBasePath" is not allowed to be empty`

![Screen Shot 2020-08-13 at 19 08 06](https://user-images.githubusercontent.com/29503118/90164998-5b045700-dd98-11ea-81ba-8a1fb6d735a4.png)

I had two .items with `activeBasePath`, both were empty string quotes, and I commented both of them out.

5. I ran `yarn start again`. This time it seemed to run, but I got the following errors, in my terminal:

![Screen Shot 2020-08-13 at 19 11 54](https://user-images.githubusercontent.com/29503118/90165384-dfef7080-dd98-11ea-98c0-702dfa1e91db.png)

The same error in my browser, 
```
Failed to compile
./src/theme/SearchBar/index.js
Module not found: Can't resolve 'docsearch.js' in '/Users/david/repos/erxes/docs/src/theme/SearchBar' 
```

Note:

- Everything was working great before the upgrade, but I wanted to upgrade
- I have done the above steps 5 times with exactly the same results
- I searched the docs, github and google and cannot find any reference to this issue. I suspect this is something I am doing wrong, and not necessarily a docusaurus issue.

## Your Environment

- Docusaurus version used: **upgrading from version 2 alpha.55 to alpha.61**
- Environment name and version (e.g. Chrome 78.0.3904.108, Node.js 10.17.0): **on a Mac with latest Node**

## Reproducible Demo

This is the link to the repo with the above issue. 

https://github.com/erxes/erxes/tree/docusaurus-upgrade-error 

(Note: It is part of a larger mono-repo for erxes.)

Thank you.

**Comment by anshulrgoyal** [comment]
I am sorry for so many errors but it is not good practise to have an empty string without a reason.
- `homePageId` is the `id` of docs which is displayed at `docs/`
- `activeBasePath` will mark the item active

Try to update `@docusaurus/theme-search-algolia` plugin.

**Comment by anshulrgoyal** [comment]
@davidkartuzinski New release contains an update to algolia search plugin and due to your website swizzles searchBar it is broken.

**Comment by anshulrgoyal** [comment]
If u just delete it. It should work fine

**Comment by anshulrgoyal** [comment]
![image](https://user-images.githubusercontent.com/32068075/90175144-9c126080-ddc4-11ea-84a9-c122028a28cd.png)

**Comment by davidkartuzinski** [comment]
@anshulrgoyal - No problem about the errors!  They makes sense. I understand this is alpha, and it is already so polished with in terminal messages and instructions! So great. I consider this issue my ignorance, not anything "wrong" with docusaurus.

Your instructions helped me to understand what was happening. What I did to fix this issue was to go into the node_modules/@docusaurus/ and delete the @docusaurus/theme-search-algolia, then at the terminal I just simply ran `@docusaurus/theme-search-algolia` and everything now works.

Again thank you for your incredibly fast response. This issue is resolved for me.
